services/ui-auth/libs/users.json

```diff
@@ -303,15 +303,15 @@
     ]
   },
   {
-    "username": "cypressadminuser@test.com",
+    "username": "e2eadminuser@test.com",
```

Comment:
I guess these get fixed when we deploy again so that the dynamo tables have them in there? Pardon my ignorance.

Reply:
Good question.  I know on new environments the create users bootstrap function will create the users based on this file, but I'm not sure about when we update an existing environment.  I'll have to check after we merge to main.  I'm expecting we may need to trigger the bootstrap users function manually.